line-bot-servlet/src/test/java/com/linecorp/bot/servlet/LineBotCallbackRequestParserTest.java

```diff
@@ -52,9 +61,11 @@
     public void before() throws IOException {
         when(response.getWriter())
                 .thenReturn(mock(PrintWriter.class));
+        final ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
The following code will fail when we pass a real data (See LineBotCallbackRequestParserTest#testCallRequest).
Because it doesn't set `FAIL_ON_UNKNOWN_PROPERTIES` `false`.
```
new LineBotCallbackRequestParser(client, new ObjectMapper());
```
I'll send another PR for this later.